algorithms/dqn/networks.py

```diff
@@ -70,6 +70,73 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x
 
 
+class CategoricalDuelingMLP(MLP):
+    """Multilayer perceptron with dueling construction."""
+
+    def __init__(
+        self,
+        input_size: int,
+        action_size: int,
+        hidden_sizes: list,
+        atom_size: int = 51,
+        v_min: int = -10,
+        v_max: int = 10,
+        hidden_activation: Callable = F.relu,
```

Comment:
```suggestion
        hidden_activation: Callable[[torch.Tensor], torch.Tensor] = F.relu,
```

Reply:
I will open an issue for it. Thanks.

Reply:
I opened an issue: https://github.com/medipixel/rl_algorithms/issues/117